apps/hannk/benchmark.cpp

```diff
@@ -41,7 +41,7 @@ void run_benchmark(const std::string &filename, const InterpreterOptions &option
 
 }  // namespace hannk
 
-int main(int argc, char **argv) {
+__attribute__((visibility("default"))) int main(int argc, char **argv) {
```

Comment:
Why is this change needed?

Reply:
On Hexagon, this is getting compiled as a shared object, not an executable. So I'm guessing this makes main visible to dlsym.

Reply:
Right